providers/azureblob/src/main/java/org/jclouds/azureblob/blobstore/AzureBlobRequestSigner.java

```diff
@@ -35,22 +36,24 @@
 import org.jclouds.http.Uris;
 import org.jclouds.http.options.GetOptions;
 import org.jclouds.javax.annotation.Nullable;
-
 import com.google.common.base.Supplier;
 import com.google.common.net.HttpHeaders;
 import com.google.inject.Provider;
 
+
 @Singleton
 public class AzureBlobRequestSigner implements BlobRequestSigner {
-   private static final int DEFAULT_EXPIRY_SECONDS = 15 * 60;
-   private static final String API_VERSION = "2017-04-17";
+   private static final int DEFAULT_EXPIRY_SECONDS = 300 * 60;
+   private static final String API_VERSION = "2018-03-28";
```

Comment:
Why do we need to change the API version?

Reply:
true, as the API version does not have any influence on SAS. I've put back the old one